src/containers/background/background.js

```diff
@@ -18,7 +18,10 @@ store.dispatch(installed ? hydrateState() : setupExtension())
 Interface.onUpdateAvailable(() => Interface.reloadExtension())
 
 Interface.setUninstallUrl('https://getpocket.com/chrome-exit-survey/')
-
+Interface.setToolbarIcon(null, 'browser-action-icon')
+Interface.onTabCreated(({ id: tabId }) => setTimeout(() => {
+  Interface.setToolbarIcon(tabId, 'browser-action-icon')
+}, 200))
```

Comment:
Never been a fan of timeouts use as a buffer. They're problematic due to various performance issues that may supersede the intended delay.

What problem is the delay intended to solve?

Reply:
If we call setToolbarIcon without a delay it will not be applied as this will occur prior to chrome setting the default toolbar icon, so chrome will override the darkmode icon with the default icon (lightmode icon). The delay let's us wait for chrome to set the default icon and then override it. @collectedmind